crates/cxx-qt-gen/src/generator/rust/property/getter.rs

```diff
@@ -43,6 +44,13 @@ pub fn generate(
                     }
                 }
             },
+            quote! {
+                impl #cpp_class_name_rust {
+                    pub unsafe fn #getter_mutable_rust<'a>(mut self: Pin<&'a mut Self>) -> &'a mut #ty {
+                        unsafe { &mut self.rust_mut().get_unchecked_mut().#ident }
```

Comment:
Do you need the `unsafe` block inside the function when the whole function is marked `unsafe`?

Reply:
Seems you can, i thought it'd just mark the function unsafe to call, but seems also the contents is unsafe.